src/lib/hash.c

```diff
@@ -80,9 +80,9 @@
 #include <stdio.h>
 
 static const struct hash_alg_map_t {
-    pgp_hash_alg_t  type;
-    const char      *name;
-    const char      *botan_name;
+    pgp_hash_alg_t type;
+    const char *   name;
```

Comment:
Previous code was correct

Reply:
Does it actually comes from clang-format?

Reply:
Yeah, this and the following changes were introduced by clang-format, and I mislooked it. Will revert.

Reply:
...will revert, but not this one - it seems it wasn't clang-formatted at some point,  * should be one space after the type.

Reply:
yeah, ok I see. I think clang-format should put the ``*`` before the variable really. anyways, that's find